src/pages/home/home.scss

```diff
@@ -3,4 +3,10 @@ page-home {
         height: 100%;
         width: 100%;
     }
+    #profile-icon {
+        margin: 10px;
+        height: 40px;
+        width: 40px;
+        border-radius: 50%;
```

Comment:
The profile icon should not be a circle, it should be a rounded-edged square. Maybe just adding 4px of border-radius will do the trick?